airgun/entities/host.py

```diff
@@ -324,6 +320,10 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+        host_view = NewHostDetailsView(self.parent.browser)
+        host_view.wait_displayed()
+        host_view.dropdown.wait_displayed()
```

Comment:
should be part of the entities actions methods, not sure why they being added part of navigation steps

Reply:
Details navigation is in multiple places, rather than change it everywhere, I change it navigation section.

It's because we navigate via New UI to the old UI, and the entities action methods in `entities/host.py` should navigate user to old UI. 

I understand that it is not used the way it was designed but on the other hand, if it fulfill the purpose, and this  usage is something temporal, why not.